lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -86,8 +86,6 @@
 #define TEE_STORAGE_PRIVATE_REE	 0x80000000
 /* Storage is the Replay Protected Memory Block partition of an eMMC device */
 #define TEE_STORAGE_PRIVATE_RPMB 0x80000100
-/* Storage is provided by a SQLite database in the normal world filesystem */
-#define TEE_STORAGE_PRIVATE_SQL  0x80000200
```

Comment:
I think we should reserve the value so it is not re-used for some other FS later, while still allowing older TAs to be compiled with a newer dev kit. It is easy enough to do: how about:
```C
#define TEE_STORAGE_PRIVATE_SQL \
        _Pragma("GCC warning \"'TEE_STORAGE_PRIVATE_SQL' is deprecated \
(not supported by OP-TEE > 2.4.x)\"") 0x80000200
```